packages/components/src/components/modal/_modal.scss

```diff
@@ -31,21 +31,18 @@
     justify-content: center;
     width: 100vw;
     height: 100vh;
-    background-color: transparent;
+    background-color: $overlay;
     visibility: hidden;
     opacity: 0;
-    transition: background-color $duration--slow-02 motion(exit, expressive),
-      opacity $duration--moderate-02 motion(exit, expressive),
+    transition: opacity $duration--moderate-02 motion(exit, expressive),
       visibility 0ms linear $duration--moderate-02;
     content: '';
 
     &.is-visible {
-      background-color: $overlay;
+      // background-color: $overlay;
```

Comment:
Is this needed?

Reply:
depending on the visual review but if the new as-is is fine then it can be removed